src/CodiceFiscale/Calculator.php

```diff
@@ -136,7 +138,7 @@ private function calculateBirthDateAndGender()
     /**
      * Calculate the Belfiore code.
      *
-     * @returns Returns the Belfiore code.
+     * @return string Returns the Belfiore code.
```

Comment:
You could remove **Returns** here and use an uppercase **T** for the initial **the**.